doc/guides/collaborator-guide.md

```diff
@@ -210,20 +210,30 @@ All fixes must have a test case which demonstrates the defect. The test should
 fail before the change, and pass after the change.
 
 All pull requests must pass continuous integration tests. Code changes must pass
-on [project CI server](https://ci.nodejs.org/). Pull requests that only change
+on [project CI server](). Pull requests that only change
 documentation and comments can use GitHub Actions results.
 
 Do not land any pull requests without a passing (green or yellow) CI run.
-For documentation-only changes, GitHub Actions CI is sufficient.
-For all other code changes, Jenkins CI must pass as well. If there are
-Jenkins CI failures unrelated to the change in the pull request, try "Resume
-Build". It is in the left navigation of the relevant `node-test-pull-request`
-job. It will preserve all the green results from the current job but re-run
-everything else. Start a fresh CI if more than seven days have elapsed since
-the original failing CI as the compiled binaries for the Windows and ARM
-platforms are only kept for seven days.
-
-#### Useful CI jobs
+A green GitHub Actions CI result is required. A passing
+[Jenkins CI](https://ci.nodejs.org/) is also required if PR contains changes
+on files in one of the following folders (except it only changes comments):
+- `deps/`
+- `lib/`
+- `src/`
+- `test/`
```

Comment:
@richardlau would this list cover everything?

```suggestion
* `deps/`
* `lib/`
* `src/`
* `test/`
* `tools/code_cache/`
* `tools/gyp/`
* `tools/icu/`
* `tools/inspector-protocol/`
* `tools/msvs/`
* `tools/snapshot/`
* `./` (root of the repository)
```

Reply:
No, it would miss things like (not exhaustive) `tools/build-addons.js`, `tools/genv8constants.py`, `tools/js2c.py`, `tools/test.py`, `tools/v8_gypfiles`